src/markdown-pages/build-apps/build-visualization.mdx

```diff
@@ -0,0 +1,194 @@
+---
+path: '/build-apps/build-visualization'
+duration: '15 min'
+title: 'Build a custom visualization for dashboards'
+template: 'GuideTemplate'
+description: 'Build and publish a visualization'
+resources:
+  - title: New Relic One VSCode extension
+    url: https://marketplace.visualstudio.com/items?itemName=new-relic.nr1
+  - title: 'Build on New Relic One'
+    url: https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application
+  - title: New Relic VSCode extension pack
+    url: https://marketplace.visualstudio.com/items?itemName=new-relic.new-relic-extension-pack
+tags:
+  - nr1 cli
+  - NR One Catalog
+  - Subscribe visualizations
+---
+
+<Intro>
+
+You can build your own visualizations for New Relic One dashboards. This allows you and your organization to query any data sources and display them on your dashboards. In this guide, you'll learn to generate a visualization boilerplate and run it locally in New Relic One's Visualization Explorer where you can quickly test and iterate before publishing.
```

Comment:
Suggested revision for intro sentence: "Building your own visualizations for New Relic dashboards enables you and your organization to query data from any source and display it on your dashboard." (This helps to avoid a kind of repeat of the title, and also helps emphasize "any source." )

Also, when you say generate a boilerplate, I'm not sure if that's just for practice, or if that's what someone would normally create.

Reply:
I like the suggested intro sentence. Thank you!

I'm certainly open to other wording than "boilerplate." It isn't intended to be just for practice. It is referencing the folders and files that get created anytime `nr1 create --type visualization` is ran which reduces the amount of time to get a new visualization started. It provides a starting point for the developer even if they are going to completely wipe out the contents of the index.json file to start from scratch.